profiler/src/ProfilerEngine/Datadog.Profiler.Native/ManagedThreadList.cpp

```diff
@@ -222,21 +222,21 @@ bool ManagedThreadList::SetThreadOsInfo(ThreadID clrThreadId, DWORD osThreadId,
     return true;
 }
 
-bool ManagedThreadList::SetThreadName(ThreadID clrThreadId, shared::WSTRING* pThreadName)
+bool ManagedThreadList::SetThreadName(ThreadID clrThreadId, const shared::WSTRING& pThreadName)
```

Comment:
Maybe rename pThreadName into threadName now that it is no more a pointer
(apply to ManagedThreadList and ManagedThreadInfo methods)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2750/commits/57c8be29c474387816ad3300189248aab62a7818